astropy/table/table.py

```diff
@@ -1,4 +1,5 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.extern import six
```

Comment:
Hmm. I thought I left a comment here about not needing `six` but it has disappeared. Please remove usage of `six`. We do not support Python 2 anymore.

Reply:
Ah, sure removed it :smile: